app/models/Errors.scala

```diff
@@ -0,0 +1,7 @@
+package models
+
+/**
+  * Error when uploading CSV
+  * @param msg the error message
+  */
+case class ErrorUploadCsv(msg: String)
```

Comment:
All case class should be final (it should have been forced in the scala language but for retro compatibility reasons they never did) (see [stack overflow](http://stackoverflow.com/questions/34561614/should-i-use-the-final-modifier-when-declaring-case-classes))